lib/fat_free_crm/permissions.rb

```diff
@@ -44,7 +44,7 @@ def #{model}_ids=(value)
               permissions_to_remove = Permission.where(
                 #{model}_id: self.#{model}_ids - value,
                 asset_id: self.id,
-                asset_type: self.class
+                asset_type: self.class.to_s
```

Comment:
Should this be to_s or class_name? I'm guessing it magically worked out as class_name previously. Lets split out and look at tests, so we can close #614 in isolation

Reply:
Nevermind, it's on a Class not an Object so boils down to the same thing.